engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -57,12 +55,16 @@ object WorkflowManagerActor {
     * Data
     */
   case class WorkflowManagerData(workflows: Map[WorkflowId, ActorRef]) {
-    def idFromActor(actor: ActorRef): Option[WorkflowId] =
-      workflows.collectFirst { case (id, a) if a == actor => id }
+    def idFromActor(actor: ActorRef): Option[WorkflowId] = workflows.collectFirst { case (id, a) if a == actor => id }
     def withAddition(entry: WorkflowIdToActorRef): WorkflowManagerData =
       this.copy(workflows = workflows + (entry.workflowId -> entry.workflowActor))
-    def without(id: WorkflowId): WorkflowManagerData =
-      this.copy(workflows = workflows - id)
+
+    def withAddition(entries: List[WorkflowIdToActorRef]): WorkflowManagerData = {
```

Comment:
I waffled on this matter so wanted to point it out. If people don't like the overloaded signature, I'm happy to call it something like `withAdditions`, collapse the single form above into this somehow, etc.